doc/source/programs/gdal_raster_scale.rst

```diff
@@ -0,0 +1,101 @@
+.. _gdal_raster_scale_subcommand:
+
+================================================================================
+"gdal raster scale" sub-command
+================================================================================
+
+.. versionadded:: 3.11
+
+.. only:: html
+
+    Scale the values of the bands of a raster dataset
+
+.. Index:: gdal raster scale
+
+Synopsis
+--------
+
+.. program-output:: gdal raster scale --help-doc
+
+Description
+-----------
+
+:program:`gdal raster scale` can be used to rescale the input pixels values
+from the range :option:`--srcmin` to :option:`--srcmax` to the range
+:option:`--dstmin` to :option:`--dstmax`. If omitted the output range is 0 to 255.
+If omitted the input range is automatically computed from the source dataset.
+This may be a slow operation on a large source dataset, and if using it multiple times
+for several gdal_translate invocation, it might be beneficial to call
+``gdal raster info --stats {source_dataset}`` priorly to precompute statistics, for
+formats that support serializing statistics computations (GeoTIFF, VRT...)
+Note that the values specified for the ranges are only used to compute a scale and
+offset to apply to the input raster values. In particular, ``srcmin`` and
+``srcmax`` are not used to clip input values unless :option:`--exponent`
+is also specified.
+Instead of being clipped, source values that are outside the range of ``srcmin``
+and ``srcmax`` will be scaled to values outside the range of ``dstmin`` and ``dstmax``.
+If clipping without exponential scaling is desired,
+``--exponent 1`` can be used.
+
```

Comment:
Warn somewhere that the scaled values may not fit into the original datatype. Because values are not clipped the overshoot may surprise users.

Reply:
note added